consensus/parlia/abi.go

```diff
@@ -1391,1733 +1391,3913 @@ const validatorSetABIBeforeLuban = `
-        "name":"misdemeanor",
-        "outputs":[
-
-        ],
-        "stateMutability":"nonpayable",
-        "type":"function"
-    },
-    {
-        "inputs":[
-
-        ],
-        "name":"numOfCabinets",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "inputs":[
-
-        ],
-        "name":"numOfJailed",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "inputs":[
-
-        ],
-        "name":"numOfMaintaining",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "inputs":[
-
-        ],
-        "name":"previousHeight",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "inputs":[
-
-        ],
-        "name":"totalInComing",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "inputs":[
-            {
-                "internalType":"string",
-                "name":"key",
-                "type":"string"
-            },
-            {
-                "internalType":"bytes",
-                "name":"value",
-                "type":"bytes"
-            }
-        ],
-        "name":"updateParam",
-        "outputs":[
-
-        ],
-        "stateMutability":"nonpayable",
-        "type":"function"
-    },
-    {
-        "inputs":[
-            {
-                "internalType":"uint256",
-                "name":"",
-                "type":"uint256"
-            }
-        ],
-        "name":"validatorExtraSet",
-        "outputs":[
-            {
-                "internalType":"uint256",
-                "name":"enterMaintenanceHeight",
-                "type":"uint256"
-            },
-            {
-                "internalType":"bool",
-                "name":"isMaintaining",
-                "type":"bool"
-            },
-            {
-                "internalType":"bytes",
-                "name":"voteAddress",
-                "type":"bytes"
-            }
-        ],
-        "stateMutability":"view",
-        "type":"function"
-    },
-    {
-        "stateMutability":"payable",
-        "type":"receive"
-    }
+  {
```

Comment:
Are there any incompatibilities between the new ABI and the old ABI?

If the answer is yes, then we can not change this in-place, we need to have two ABI at the same time.

Like what we have now "validatorSetABIBeforeLuban".

Reply:
There's no incompatibilities